media/css/mozorg/contact-spaces.less

```diff
@@ -437,6 +437,11 @@
     }
 }
 
+/* override float rule for contact us tab */
+.entry.contact-us {
+    float: none;
```

Comment:
The `.clearfix` on the parent `.container` is saving us from layout issues. If we had content underneath "Questions & feedback" however, there would be issues. I tend to err on the side of clearing floats. Doesn't hurt anything. :wink: